presto-orc/src/main/java/com/facebook/presto/orc/cache/OrcCacheConfig.java

```diff
@@ -35,6 +35,9 @@
     private DataSize stripeStreamCacheSize = new DataSize(0, BYTE);
     private Duration stripeStreamCacheTtlSinceLastAccess = new Duration(0, SECONDS);
 
+    private boolean dwrfStripeCacheEnabled;
+    private DataSize expectedFileTailSize = new DataSize(OrcFileTailSource.EXPECTED_FOOTER_SIZE_BYTES, BYTE);
```

Comment:
Can we static important this, so that we can directly use `EXPECTED_FOOTER_SIZE_BYTES`.